### DISCUSSION TITLE
Beginner question regarding layer aggregation networks

### DISCUSSION THREAD

**Discussion by Desperado17** [ultralytics/ultralytics#15631] [General]
Greetings,

as a beginner with neural networks I'm looking for a documentation what exactly is the purpose of components like ELAN and its variants in the neck part of an image recognition neural network. Can someone point me towards a document?

Regards

**Comment by glenn-jocher** [comment]
@Desperado17 greetings,

For a detailed understanding of ELAN and its variants in the neck part of neural networks, I recommend reviewing the YOLOv8 documentation and related research papers. These resources provide comprehensive insights into the architecture and its components.

Regards

**Comment by Desperado17** [comment]
@glenn-jocher Is there are a single document for YOLOv8 documentation?

**Reply by glenn-jocher** [reply]
@Desperado17 you can find the comprehensive YOLOv8 documentation at https://docs.ultralytics.com. This resource includes detailed information on various aspects of YOLOv8, including installation, usage, and model specifics.

**Comment by Desperado17** [comment]
@glenn-jocher I think I need a book about neural network fundamentals. Can you recommende me something that is specifically suited towards Yolo?

**Reply by glenn-jocher** [reply]
@Desperado17 for a solid understanding of neural networks with a focus on YOLO, I recommend starting with "Deep Learning" by Ian Goodfellow, Yoshua Bengio, and Aaron Courville. While it doesn't focus exclusively on YOLO, it provides a comprehensive foundation in neural networks. For YOLO-specific insights, the official YOLO documentation and research papers are invaluable resources.
